ddtrace/contrib/trace_utils.py

```diff
@@ -106,12 +107,46 @@ def ext_service(pin, config, default=None):
     return default
 
 
-def set_http_meta(span, config, method=None, url=None, status_code=None):
+def get_error_codes():
+    error_codes = []
+    try:
+        error_str = config.http_server.error_statuses
```

Comment:
Is this new functionality? If so I think we should move it to another PR.

Reply:
Not new but not widely implemented. Here's some [documentation](https://github.com/DataDog/architecture/blob/master/rfcs/apm/integrations/status-error-mapping/rfc.md) on it.